common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -25,6 +25,7 @@
             var match;
 
             this.$el = options.el;
+            this.showThreadPreview = true;
```

Comment:
I agree. I don't think we need to be able to control the preview on this view, it should always be on.